disconnected/mirroring/about-installing-oc-mirror-v2.adoc

```diff
@@ -79,7 +79,7 @@ include::modules/oc-mirror-disk-to-mirror-v2.adoc[leveloffset=+2]
 [id="additional-resources"]
 == Additional resources
 
-* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-osus[Updating a cluster in a disconnected environment using the OpenShift Update Service].
+* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-OSUS[Updating a cluster in a disconnected environment using the OpenShift Update Service].
```

Comment:
Does the switch to uppercase `-OSUS` need to happen for build reasons? Could instead the `id` in the `disconnected-update-osus.adoc` module be changed to lowercase to match the other way?

Reply:
I think your suggestion makes more sense. Thanks!